pkg/function/restore_rds_snapshot.go

```diff
@@ -117,6 +112,9 @@ func (*restoreRDSSnapshotFunc) Exec(ctx context.Context, tp param.TemplateParams
 		if err := Arg(args, RestoreRDSSnapshotDBEngine, &dbEngine); err != nil {
 			return nil, err
 		}
+		if err := Arg(args, RestoreRDSSnapshotNamespace, &namespace); err != nil {
+			return nil, err
+		}
```

Comment:
We removed the namespace from requirement args right? do we still need this check, or we should have `OptArg`

Reply:
Arg is required if snapshotID is nil